src/dev/i18n/utils.js

```diff
@@ -38,6 +38,12 @@ export const globAsync = promisify(glob);
 export const makeDirAsync = promisify(fs.mkdir);
 export const accessAsync = promisify(fs.access);
 
+export function arraysDiff(left = [], right = []) {
```

Comment:
I would just implement `difference` function and call it 2 times.
```js
// instead of
const [addedMessages, removedMessages] = arraysDiff(defaultMessagesIds, cachedMessagesIds);

// this
const addedMessages = difference(defaultMessagesIds, cachedMessagesIds);
const removedMessages = difference(cachedMessagesIds, defaultMessagesIds);
```

Or just use [lodash difference](https://lodash.com/docs/4.17.10#difference)
What do you think?